src/l3.c

```diff
@@ -342,6 +372,29 @@ l3_log_mmap(const char *msg, const uint64_t arg1, const uint64_t arg2,
     l3_log->slots[idx].arg2 = arg2;
 }
 
+/**
+ * l3_log_write() - 'C' interface to log L3 log-entries using write()
+ */
+void
+l3_log_write(const char *msg, const uint64_t arg1, const uint64_t arg2)
+{
+    L3_ENTRY le     = {0};
+    L3_ENTRY *lep   = &le;
+
+    lep->tid = l3_mytid();
+
+    lep->msg = msg;
+    lep->arg1 = arg1;
+    lep->arg2 = arg2;
+
+    if (write(l3_log_fd, lep, sizeof(*lep) != sizeof(*lep))) {
```

Comment:
@gapisback I think this line has a bug - the closing parens at the end are out of order, so it always gives `len` argument to `write()` of 0.

Reply:
Yes! You are right. Thanks, for catching it.

Will fix ... and re-measure overall throughput with write.

I'll see if I can whip-up a simple unit-test to exercise these new interfaces and verify that the file-size is coming out as expected.

If that's taking too much time, I'll punt and open an issue for a follow-through.